messages/de/SpaceModule.base.php

```diff
@@ -106,7 +106,7 @@
   'Shown on About Page.' => 'Wird auf Informationsseite angezeigt.',
   'Sorry, you are not allowed to leave this space!' => 'Sorry, aber du kannst diesen Space nicht verlassen.',
   'Sorting' => 'Sortierung',
-  'Space' => 'Space',
+  'Space' => 'Gruppen und Vereine',
```

Comment:
space is singular, do you think this will match?

Reply:
How can I answer this question?

Reply:
i guess "Spaces" plural would translate to "Gruppen und Vereine" but "Space" is something singular, so it would not fit in HumHub having it "Gruppen und Vereine".

Reply:
singular -> Gruppe/Verein

Reply:
doen in 4a99163619255550b92deb0f6f6fe74607d5c22b